src/cmd-build

```diff
@@ -143,6 +148,12 @@ if [ $# -eq 0 ]; then
     set -- qemu
 fi
 
+if [ -z "${VERSION}" ] && [ -n "${VERSIONARY}" ]; then
+    # let error out if file does not exist
```

Comment:
```suggestion
    # let's error out if file does not exist
```

Reply:
It's more like, "let it error out if the file does not exist". I won't fix it to not ruin CI, but I'll queue it up for a future cosa PR.